core/displaymodels/src/commonMain/kotlin/com/adammcneilly/pocketleague/core/displaymodels/SwissTeamResultDisplayModel.kt

```diff
@@ -13,3 +15,22 @@ data class SwissTeamResultDisplayModel(
     val overline: String,
     val subtitle: String,
 )
+
+fun SwissTeamResult.toDisplayModel(): SwissTeamResultDisplayModel {
+    val gameRecord = "${this.gameWins} - ${this.gameLosses}"
+
+    val gameDifferential = this.gameWins - this.gameLosses
+    val gameDifferentialString = if (gameDifferential > 0) {
+        "+$gameDifferential"
+    } else {
+        gameDifferential.toString()
+    }
+
+    val matchDifferential = "${this.matchWins} - ${this.matchLosses}"
+
+    return SwissTeamResultDisplayModel(
+        team = this.team.toOverviewDisplayModel(),
+        overline = "TODO",
```

Comment:
Remove overline? Or maybe call it something else, could put it elsewhere on the card.